### PR TITLE
Feature/LPFG-13 - Adding cookie page and banner

### PR DESCRIPTION
**Pull request by m24murray**
- adding cookies page and banner (cookie page has fairly basic styling for now, but may need stylised in line with gds in future)
- adding cookie controller to set cookie seen_cookie_message value to true
- js to check value of cookie and hide banner accordingly
- if user's browser has javascript turned off, the message should always appear

### CODE REVIEW

File: src/ui/component/Page.html [hunk 1]
```diff
@@ -14,6 +14,12 @@
     <script async src="https://www.googletagmanager.com/gtag/js?id={{config.GOOGLE_ANALYTICS_ID}}"></script> {{{config.GOOGLE_ANALYTICS_CODE}}}
 </head>
 <body>
+    <div id="global-cookie-message">
+        <p>
+            GOV.UK uses cookies to make the site simpler.
```
Comment (superafroman) [inline src/ui/component/Page.html]: Probably need to review this message - can you run past Rich?

File: src/ui/server.ts [hunk 2]
```diff
@@ -122,7 +141,8 @@ app.use(
 app.use(serveStatic('assets'))
 app.use(favicon(path.join('assets', 'img', 'favicon.ico')))
 
-passport.configure('lpg-ui', config.AUTHENTICATION.serviceUrl, app)
+passport.configure('lpg-ui', config.AUTHENTICATION.	serviceUrl, app)
```
Comment (superafroman) [inline src/ui/server.ts]: tab/space before `serviceUrl`?

File: src/ui/component/Page.html (hunk 1 above)
Comment (m24murray) [reply src/ui/component/Page.html]: Changing to Civil Service Learning

**Comment by m24murray** [comment]
This should be ready to go now, if this gets out of date and i'm not around can someone else rebase and push? 🙏🛠👉

**Pull request merged**